messages/it.json

```diff
@@ -1,63 +1,63 @@
 {
   "pickupPointsModal.loading": "Caricamento in corso…",
-  "pickupPointsModal.deliver": "Consegnare",
-  "pickupPointsModal.pickup": "Ritirare",
-  "pickupPointsModal.items": "{itemsAmount, plural, =0 {} one {# articolo} other {# articoli}}",
-  "pickupPointsModal.cancelBackList": "Vedi tutti i punti di ritiro",
+  "pickupPointsModal.deliver": "Consegna",
+  "pickupPointsModal.pickup": "Ritiro",
+  "pickupPointsModal.items": "{itemsAmount, plural, =0 {} one {# oggetto} other {# oggetti}}",
+  "pickupPointsModal.cancelBackList": "Visualizza tutti i punti di ritiro",
   "pickupPointsModal.slaOption": "{name} - {price} - {shippingEstimate}",
-  "pickupPointsModal.free": "Gratuito",
+  "pickupPointsModal.free": "Gratis",
   "pickupPointsModal.selectPickupPoint": "Seleziona un punto di ritiro",
-  "pickupPointsModal.pointDetails": "Dettagli del punto di ritiro",
-  "pickupPointsModal.closeButton": "Chiude",
+  "pickupPointsModal.pointDetails": "Dettagli relativi al punto di ritiro",
+  "pickupPointsModal.closeButton": "chiudi",
```

Comment:
shouldn't this remain capitalized?

Reply:
Checking with Vlad

Reply:
All good, the reference is the english file, which is `close`.